### DISCUSSION TITLE
BetterSoftware.it, sustainable IT conference in Italy

### DISCUSSION THREAD

**Discussion by fullo** [Green-Software-Foundation/discussions#160] [Events]
In October 11th I will help to organize in Bologna, Italy, BetterSoftware.it conference. The conference is organized by GrUSP.org a non-profit organization that works to create a better ecosystem in the web development scene since 2001. BetterSoftware will have talks about Sustainable IT and Green Software. 

This first edition will be in Italian, but I plan to organize an international 2nd edition in 2025. 

The call for papers is open and I'd love to have some partnerships with the GSF to gain some visibility outside GrUSP's usual channels.

ciao
F.

**Comment by florent-morel** [comment]
Excellent initiative @fullo !
Could you please share the URL?
As being close to Italy (French Riviera), this might be an interesting conference for us (once international as we have staff from all over the world).

**Comment by fullo** [comment]
@florent-morel the website is https://www.bettersoftware.it, the call for paper is still open at https://sessionize.com/bettersoftware-2024/
